go/host/p2p/p2p_status.go

```diff
@@ -0,0 +1,80 @@
+package p2p
+
+import (
+	"sync"
+	"time"
+
+	hostcommon "github.com/obscuronet/go-obscuro/go/common/host"
+)
+
+const (
+	_failedMessageRead        = "Failed Message Reads"
+	_failedMessageDecode      = "Failed Message Decodes"
+	_failedConnectSendMessage = "Failed Peer Connects"
+	_failedWriteSendMessage   = "Failed Socket Writes"
+	_receivedMessage          = "Received Messages"
+)
+
+var _rollingPeriod = 5 * time.Minute
+
+type status struct {
+	lock          sync.RWMutex
+	currentStatus map[string]*ttlMap
```

Comment:
Worth a comment to explain maybe? (e.g. "this is a map of peerHostAddr -> failure metrics map")

Reply:
Adding one 👍